### PR TITLE
fix: add missing optional keyword to protobuf syntax

### PR DESCRIPTION
**Pull request by yusdacra**
It appears I forgot about the "optional" keyword while doing the other PR. This adds it.

### CODE REVIEW

File: runtime/queries/protobuf/highlights.scm [hunk 1]
```diff
@@ -23,7 +24,6 @@
 
 [
   (mapName)
-  (oneofName)
```
Comment (pickfire) [inline runtime/queries/protobuf/highlights.scm:26]: Why is this removed?

**Comment by pickfire** [comment]
Can you please send a pull request back to upstream?

File: runtime/queries/protobuf/highlights.scm (hunk 1 above)
Comment (yusdacra) [reply runtime/queries/protobuf/highlights.scm:26]: I forgot to remove it apparently. One ofs aren't types, they are represented as fields so removing it makes the highlighting make more sense.

**Comment by yusdacra** [comment]
> Can you please send a pull request for your changes back to upstream https://github.com/nnnewb/tree-sitter-protobuf?

I will try to make a PR hopefully.
